src/components/fields/ObjectField.js

```diff
@@ -81,7 +82,7 @@ if (process.env.NODE_ENV !== "production") {
     onChange: PropTypes.func.isRequired,
     formData: PropTypes.object,
     required: PropTypes.bool,
-    SchemaField: PropTypes.func.isRequired,
+    registry: PropTypes.object,
```

Comment:
Would be great if we could describe a little more what's expected as a data structure here, eg. expected keys and their type.